dll/cpl/appwiz/appwiz.h

```diff
@@ -17,6 +17,10 @@
 #include <winreg.h>
 #include <winnls.h>
 #include <shlobj.h>
+#include <intshcut.h>
+#include <shlwapi.h>
+#include <tchar.h>
```

Comment:
tchar.h should not be needed.

Reply:
<strsafe.h> requests <tchar.h> inclusion. It's a necessary.

Reply:
ah ok then, sorry :)